guardian/pan-domain-authentication
# Update google-api-client to 2.x

## What's changed?

Upgrades `google-api-services-admin-directory` to latest, which pulls in version `2.1.2` of `google-api-client`.  

## How to test

Publish the library locally with `sbt "+publishLocal"`, and bump to the snapshot version in a consuming project that issues pan-domain-auth tokens – [Typerighter's](https://github.com/guardian/typerighter) rule manager project is one example. Log in as a new user. Does authentication work as expected?
